org.adempiere.base/src/org/compiere/db/AdempiereDatabase.java

```diff
@@ -344,9 +344,10 @@ public Connection getDriverConnection (String dbUrl, String dbUid, String dbPwd)
 	/**
 	 * @param columnName
 	 * @param csv comma separated value
+	 * @param isNotClause
 	 * @return subset sql clause
 	 */
-	public String intersectClauseForCSV(String columnName, String csv);
+	public String intersectClauseForCSV(String columnName, String csv, boolean isNotClause);

```

Comment:
for backward compatibility, please also keep the existing method signature.

Reply:
done